### PR TITLE
Updated featured with index and relevant changes included

### PR DESCRIPTION
**Pull request by gsalunke**
image
added fonts into featured components

### CODE REVIEW

File: FPM.ftd [hunk 1]
```diff
@@ -59,6 +59,7 @@ email: arpita@fifthtry.com
   - Resumes: /featured/resumes/
   - Color Schemes: /featured/color-cs/
   - Component Libraries: /featured/components-library/
+  - How to Use Fonts: /featured/how-to-use-fonts/
```
Comment (amitu) [inline FPM.ftd:62]: No, this folder is for featured stuff. This is a how to. It should go in the how-to folder.

Comment (amitu) [reply FPM.ftd:62]: Make the url `fpm.dev/how-to/custom-fonts`.

Comment (gsalunke) [reply FPM.ftd]: @amitu issue is fixed! Please, review again!

Comment (amitu) [reply FPM.ftd]: The URL is not `fpm.dev/how-to/custom-fonts/`. The other document should have the URL: `fpm.dev/how-to/create-font-package/`. 

Move both the how-to documents in `how-to` folder.

**Pull request merged**